index.js

```diff
@@ -0,0 +1,125 @@
+const express = require('express');
+const helmet = require('helmet');
+const cors = require('cors');
+const server = express();
+const session = require('express-session'); // session library to handle cookies and authentication
+const KnexSessionStore = require('connect-session-knex')(session); // knex sqlite db support for express-session
+
+const bcrypt = require('bcryptjs');
+server.use(helmet());
+server.use(cors());
+server.use(express.json());
+const db = require('./data/dbConfig.js'); // database configuration file location
+
+// CONFIG: server settings
+const serverPort = 7100; // server port
+const serverName = `auth-i`; // Name of server to display at "/" endpoint 
+const serverRepo = `https://github.com/LambdaSchool/auth-i/pull/341`;
+
+// CONFIG: endpoint routing
+// const users = require('./data/routes/usersRoutes');
+// server.use('/api/users', users);
+
+// CONFIG: Models
+const users = require('./data/models/usersModels');
+
+// CONFIG: express-session
+const sessionConfig = {
+  secret: 'its.a.secret.to.everybody',
+  name: 'lambda', // defaults to connect.sid
+  httpOnly: true, // JS cannot access this session
+  resave: false,
+  saveUninitialized: false, // Legally obligated!
+  cookie: {
+    secure: false, // over https false
+    maxAge: 1000 * 60 * 1 // 1000 ms * 60 seconds * 1 minute 
+  },
+  store: new KnexSessionStore({
+    tablename: 'sessions', // this is the tablename of the session store
+    sidfieldname: 'sid', // this is the field name that stores the session id
+    knex: db, // name of configured knex database
+    createtable: true, // this option creates the table automatically if it doesn't exist
+    clearInterval: 1000 * 60 * 60, // removes only expired sessions
+  })
```

Comment:
Great comments on the new material. This shows that you really know what your code is doing.